code/game/gamemodes/gang/gang_datum.dm

```diff
@@ -53,36 +94,56 @@
 		switch(color)
 			if("red")
 				color_hex = "#DA0000"
+				inner_outfit = pick(text2path("/obj/item/clothing/under/color/[color]"),/obj/item/clothing/under/lawyer/red)
 			if("orange")
 				color_hex = "#FF9300"
+				inner_outfit = pick(text2path("/obj/item/clothing/under/color/[color]"),/obj/item/clothing/under/geisha)
 			if("yellow")
 				color_hex = "#FFF200"
+				inner_outfit = pick(text2path("/obj/item/clothing/under/color/[color]"),/obj/item/clothing/under/burial)
 			if("green")
 				color_hex = "#A8E61D"
+				inner_outfit = pick(text2path("/obj/item/clothing/under/color/[color]"),/obj/item/clothing/under/syndicate/camo ,/obj/item/clothing/under/suit_jacket/tan)
 			if("blue")
 				color_hex = "#00B7EF"
+				inner_outfit = pick(text2path("/obj/item/clothing/under/color/[color]"),/obj/item/clothing/under/suit_jacket/navy)
 			if("purple")
 				color_hex = "#DA00FF"
+				inner_outfit = pick(text2path("/obj/item/clothing/under/color/lightpurple"),/obj/item/clothing/under/lawyer/purpsuit)
+			if("white")
+				color_hex = "#FFFFFF"
+				inner_outfit = pick(text2path("/obj/item/clothing/under/color/[color]"),/obj/item/clothing/under/suit_jacket/white)
```

Comment:
don't use text paths, they compile even when the path doesn't exist anymore, doesn't make any sense, just give the full path and pick it

half of them don't even insert the colour either ree